rpctest/node.go

```diff
@@ -167,17 +159,7 @@ func (n *nodeConfig) String() string {
 
 // cleanup removes the tmp data and log directories.
 func (n *nodeConfig) cleanup() error {
-	dirs := []string{
-		n.logDir,
-		n.dataDir,
-	}
-	var err error
-	for _, dir := range dirs {
-		if err = os.RemoveAll(dir); err != nil {
-			log.Printf("Cannot remove dir %s: %v", dir, err)
-		}
-	}
-	return err
+	return os.RemoveAll(filepath.Dir(n.dataDir))
```

Comment:
This entire function is no longer needed.  Tearing down the harness recursively removes the top level directory.  See `harness.TearDown`.